docs/user-guide/01-blueprint-reference.md

```diff
@@ -357,6 +357,74 @@ append = "nosmt=force"
 </TabItem>
 </Tabs>
 
+### Subscription-manager (RHSM) 🔵 {#rhsm}
+
+An optional set of configuration options for the Red Hat Subscription Manager (RHSM) client.
+
+The customization expects that `subscription-manager` package is installed in the image.
```

Comment:
For newbies maybe add a link to the package section? (no merge blocker for me though)

Reply:
Thanks, I added the link and reworded it a bit.